### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -32,104 +32,106 @@
...
+    "remark-react": "4.0.3",
+    "reselect": "3.0.1",
+    "tooltip.js": "1.2.0",
+    "topojson": "3.0.2",
+    "turf-linestring": "1.0.2",
+    "whatwg-fetch": "2.0.4"
   },
   "engines": {
-    "node": "8.2.1"
+    "node": "10.6.0"
```
Comment (sorodrigo) [inline frontend/package.json]: This should always be `>= x.x.x` to a reasonable version. And by reasonable I mean lowest working version.

Comment (tiagojsag) [reply frontend/package.json]: :+1:

**Comment by sorodrigo** [comment]
I have a lot of warnings from prettier 😞

**Comment by tiagojsag** [comment]
tomorrow I'm getting rid of prettier. I honestly see little sense in many of the "errors" it detects, and whatever time it saves us by enforcing a common code structure, it seems to require a lot more in maintenance and making sure we comply with it.

File: frontend/package.json [hunk 2]
```diff
@@ -47,17 +49,17 @@
...
-    "eslint-plugin-standard": "3.0.1",
+    "eslint-import-resolver-webpack": "0.10.1",
+    "eslint-loader": "2.0.0",
+    "eslint-plugin-import": "2.13.0",
+    "eslint-plugin-jest": "21.18.0",
+    "eslint-plugin-jsx-a11y": "6.1.1",
+    "eslint-plugin-prettier": "2.6.2",
+    "eslint-plugin-promise": "3.8.0",
+    "eslint-plugin-react": "7.10.0",
+    "eslint-plugin-standard": "3.1.0",
```
Comment (sorodrigo) [inline frontend/package.json]: not needed, we're using airbnb

**Pull request merged**